crates/symbolicator-service/src/config.rs

```diff
@@ -558,7 +558,7 @@ impl Default for Config {
             deny_list_threshold: 20,
             deny_list_block_time: Duration::from_secs(24 * 60 * 60),
             deny_list_never_block_hosts: Vec::new(),
-            max_concurrent_requests: Some(120),
+            max_concurrent_requests: Some(200),
```

Comment:
Why 200, and not 1000?

Reply:
I'd rather (nearly) double it and observe first before doing that 😅